### PR TITLE
[WIP] Laravel Blade

### PR DESCRIPTION
**Pull request by haringsrob**
This pr is just to show the concept.

It uses the output of code written in https://github.com/haringsrob/laravel-dev-tools (a lsp on its own).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -28,7 +28,8 @@
         "symfony/process": "^5.4",
         "jetbrains/phpstorm-stubs": "dev-master",
         "microsoft/tolerant-php-parser": "^0.1.0",
-        "phpactor/map-resolver": "^1.5.0"
+        "phpactor/map-resolver": "^1.5.0",
+        "illuminate/view": "^8.83"
```
Comment (haringsrob) [inline composer.json:32]: This is not needed.

File: lib/WorseReflection/Bridge/Phpactor/MemberProvider/MixinMemberProvider.php [hunk 2]
```diff
@@ -30,12 +32,25 @@ public function provideMembers(ServiceLocator $locator, ReflectionClassLike $cla
                 continue;
             }
 
+            $visited = $reflection->getClass()->getVisited();
```
Comment (haringsrob) [inline lib/WorseReflection/Bridge/Phpactor/MemberProvider/MixinMemberProvider.php:35]: Unrelated..

**Comment by dantleech** [comment]
also nice to see that the Language Server package is used by at least one other thing :smile:

**Comment by haringsrob** [comment]
I still have to work on this. Might do it later today to make it into a core extension.

**Comment by dantleech** [comment]
closing for now as stale, feel free to continue to discuss here
